src/api/app/controllers/webui/project_controller.rb

```diff
@@ -411,22 +411,6 @@ def monitor
     @repoarray = repohash.sort
   end
 
-  def toggle_watch
-    if User.current.watches?(@project.name)
-      logger.debug "Remove #{@project} from watchlist for #{User.current}"
-      User.current.remove_watched_project(@project.name)
-    else
-      logger.debug "Add #{@project} to watchlist for #{User.current}"
-      User.current.add_watched_project(@project.name)
-    end
-
-    if request.env['HTTP_REFERER']
-      redirect_back(fallback_location: root_path)
-    else
-      redirect_to action: :show, project: @project
-    end
-  end
-
```

Comment:
Please squash this commit and the following together. Since you move, and change, this code, it makes sense to keep this in one commit.

Reply:
For sure! 🙂 
I'll squash the commits when i'm done working on this feature